Amplitude/Amplitude.h

```diff
@@ -212,6 +215,18 @@
 
 /*!
  @method
+ 
+ @abstract
+ Sets a group for the user. Can be called multiple times to set multiple groupType: groupValue pairs for a user (up to 5). For example you could call this to specify that a user is in orgId 15.
+
+ @param groupType               If you are setting a group for a user, you need to specify a group type (for example orgId to indicate user is part of a specific organization).
+ @param groupName               The value of the group name (for example for groupType orgId, the groupName would be the actual id number, like 15).
+ */
+
+- (void)setGroupType:(NSString*) groupType groupName:(NSObject*) groupName;
```

Comment:
I originally had it as `setGroup`, but since the first input name is `groupType`... I guess we should make all 3 SDKs as consistent as possible.